DevCycle.SDK.Server.Local/Api/EventQueue.cs

```diff
@@ -159,7 +159,11 @@ public virtual void QueueEvent(DVCPopulatedUser user, Event @event, BucketedUser
             
             var userAndEvents = eventPayloadsToFlush[user];
 
-            userAndEvents.Events.Add(new DVCRequestEvent(@event, user.UserId, config.FeatureVariationMap));
+            var featureVars = config == null 
```

Comment:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator

This might help here.